speechbrain/utils/metric_stats.py

```diff
@@ -204,6 +210,14 @@ class ErrorRateStats(MetricStats):
         this represents character to split on after merge.
         Used with ``split_tokens`` the sequence is joined with
         this token in between, and then the whole sequence is split.
+    keep_values : bool
+        Whether to keep the values of the concepts or not.
```

Comment:
Please provide more context. For instance: " If set to True, a string containing concepts that are specified with tag_in and tag_out will be kept in the string, if False, they will be removed. This is useful for intent recognition, for instance."

Reply:
Correction done.